components/Stat.js

```diff
@@ -10,10 +10,10 @@ const StatContainer = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
-  
-  /* IE11 fix */
-  flex-shrink: 1;
+
+  flex: 1;
```

Comment:
Might be worth to ping @torkjels about this change, which he introduced in this commit only two weeks ago: https://github.com/futurice/masifunde-fe/commit/d217e63ff43785fa55b71ffe82271f366a0594bb

Reply:
So `flex: 1` is the same as 
```
flex-grow: 1;
flex-shrink: 1;
flex-basis: 0px;
```
( https://css-tricks.com/almanac/properties/f/flex/#article-header-id-5 )

Testing it out a bit now, it seems like the key to that fix was the `flex-basis`, not the `flex-shrink`.

What is this PR aimed at fixing, @denisw ?

Reply:
It fixes those stats column being rendered exceedingly narrowly.